site/pages/gc-subway-index.hbs

```diff
@@ -0,0 +1,106 @@
+---
+{
+"title": "GC-Subway Index",
+"language": "en",
+"altLangPrefix": "gc-subway-index",
+"breadcrumb": [
+{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" },
+{ "title": "Business and industry", "link": "https://www.canada.ca/en/services/business.html" },
+],
+"tag": "GCWeb",
+"dateModified": "2020-11-16",
+"share": "true"
+}
+---
+<h1 property="name" id="wb-cont" class="provisional gc-thickline">GC-Subway Index
+</h1>
+<div class="row">
+	<div class="col-md-8">
+		<div class="provisional alert alert-info">
+			<h2>This is an optional Alert</h2>
```

Comment:
The `<h2>` is pretty massive, can you add an h3 class on it to make it a tad smaller?
```suggestion
			<h2 class="h3">This is an optional Alert</h2>
```